optashift-employee-rostering-server/src/main/java/org/optaplanner/openshift/employeerostering/server/solver/WannabeSolverManager.java

```diff
@@ -49,6 +56,8 @@
 
     private ConcurrentMap<Integer, SolverStatus> tenantIdToSolverStateMap = new ConcurrentHashMap<>();
     private ConcurrentMap<Integer, Solver<Roster>> tenantIdToSolverMap = new ConcurrentHashMap<>();
+    private ConcurrentMap<Integer, Map<Shift, IndictmentView>> tenantIdToShiftIndictmentMap = new ConcurrentHashMap<>();
+    private ConcurrentMap<Integer, Map<Employee, IndictmentView>> tenantIdToEmployeeIndictmentMap = new ConcurrentHashMap<>();
```

Comment:
Race condition alert.
These are 4 different fields that on their own are thread-safe, but together they might not be.